aws/awserr/types.go

```diff
@@ -2,6 +2,16 @@ package awserr
 
 import "fmt"
 
+const (
+	// CodeSerialization is the serialization error code that is received
+	// during protocol unmarshaling.
+	CodeSerialization = "SerializationError"
```

Comment:
Hmm, that's a weird place for it.

I'll go ahead and rename this to `ErrCode*`. However, why should a response time error code live in the request package? Just for consistency?